sprint/old-sprint-guide.md

```diff
@@ -5,7 +5,7 @@
 :::
 ::::
 
-(sprint)=
+(old_sprint)=
```

Comment:
I think we can delete this file now instead of updating the target and keeping it as an inaccessible page

Reply:
Done!